lib/rules/v-on-function-call.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @author Niklas Higi
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'enforce or forbid parentheses after method calls without arguments in `v-on` directives',
+      category: undefined,
+      url: 'https://eslint.vuejs.org/rules/v-on-function-call.html'
+    },
+    fixable: 'code',
+    schema: [
+      { enum: ['always', 'never'] }
+    ]
+  },
+
+  create (context) {
+    const always = context.options[0] === 'always'
+
+    return utils.defineTemplateBodyVisitor(context, {
+      "VAttribute[directive=true][key.name='on'][key.argument!=null] VOnExpression > ExpressionStatement > *" (node) {
+        const hasParens = node.type === 'CallExpression'
+        if (hasParens && node.arguments.length > 0) return
+
+        if (always && !hasParens) {
+          context.report({
+            node,
+            loc: node.loc,
+            message: "Method calls inside of 'v-on' directives must have parentheses.",
+            fix: fixer => fixer.insertTextAfter(node, '()')
```

Comment:
Since `$event` may be required as an argument, I think that it should not be automatically fixed.

Reply:
Good point!